source/configuration_manual/authentication/ldap_conf_ext_settings.rst

```diff
@@ -0,0 +1,603 @@
+.. _authentication-ldap_conf_ext_settings:
+
+============================
+Settings in ldap conf ext
+============================
+
+This page enlists settings being configured in the ldap configuration files
+(e.g, conf.d/auth-ldap.conf.ext) for userdb, passdb or retieving sieve script
+from ldap.
+
+.. Note:: The ldap configuration files are opened as root, so should be owned by root and mode 0600.
+
+
+.. _ldap_conf_ext_setting-auth_bind:
+
+``auth_bind``
+---------------
+
+- Default: ``no``
+- Values:  :ref:`boolean`
+
+Set yes to use authentication binding for verifying password's validity.
+This works by logging into LDAP server using the username and password given by client.
+The :ref:`ldap_conf_ext_setting-pass_filter` is used to find the DN for
+the user. Note that the pass_attrs is still used, only the password field
+s ignored in it. Before doing any search, the binding is switched back
+to the default DN.
+
+.. note::
+  If you're not using authentication binding, you'll need to give
+  dovecot-auth read access to userPassword field in the LDAP server.
+  With OpenLDAP this is done by modifying /etc/ldap/slapd.conf.
+
+.. todo::
+   I omitted below for now
+   There should already be something like this:
+   access to attribute=userPassword
+   by dn="<dovecot's dn>" read # add this
+   by anonymous auth

```

Comment:
This is the old way of configuring OpenLDAP. The new way is using LDIF database (in some way I still haven't taught myself how to do). I don't think we should describe the old way.

Reply:
after searching and playing around on my test machine, updated with a way creating a .ldif file for modifying olcAccess and applying it using ldapmodify.
I am still not very sure if it is good or not, though